storyblok/field-plugin
# feat(helper): watching config files for changes

## What?
This PR created a new logic watched the `field-plugin.config.json` and prints the new sandbox URL. 


## Why?

JIRA: SHAPE-7048

## How to test? (optional)
You can use any of the templates inside `packages/cli/templates`.
1. yarn install
2. yarn dev
3. Then change the `field-plugin.config.json`
4. Observe the console where the new sandbox URL is displayed
